### PR TITLE
Step counter problem corrected on bank account step

### PR DESCRIPTION
**Pull request by PrashantMangukiya**
@TomatoToaster Pr is ready, please review it.

### Details
Step counter was not displayed on bank account steps, so corrected that problem.

### Fixed Issues
$ https://github.com/Expensify/App/issues/5580

### Tests | QA Steps

1. Go to /bank-account/new
2. Select option "Connect manually"
3. Verify the step counter in routing number/account number entry page
QA - Now Step 1 of 5 shows on bank account step.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
Web

#### Mobile Web
Mobile Web

#### Desktop
Desktop

#### iOS
iOS

#### Android
Android

### CODE REVIEW

**Comment by TomatoToaster** [comment]
Added the CP-Staging label since this resolves a deploy blocker @PrashantMangukiya thanks for getting this up quick. Reviewing and testing it now.

**Review by TomatoToaster — APPROVED** [review]

**Comment by TomatoToaster** [comment]
Looks good
image

Just waiting for E2E tests to pass before I merge this. Just note though, it will take a while 🙃

**Review by thienlnam — APPROVED** [review]

**Comment by isagoico** [comment]
image Retest was a pass!

**Pull request merged**
